src/main/java/TaskList.java

```diff
@@ -0,0 +1,120 @@
+import java.util.ArrayList;
+
+/**
+ * Represents the list of tasks in the program.
+ */
+public class TaskList {
+    protected ArrayList<Task> taskList;
+    protected ArrayList<String> fileList;
```

Comment:
I don't think the `fileList` is necessary as an instance variable, you're only using it in the constructor after all.